sources/packages/backend/apps/api/src/services/student-assessment/student-assessment.service.ts

```diff
@@ -236,4 +249,94 @@ export class StudentAssessmentService extends RecordDataModelService<StudentAsse
       .addOrderBy("assessment.submittedDate", "DESC")
       .getMany();
   }
+
+  /**
+   * Creates a manual assessment for the application.
+   * @param applicationId application id.
+   * @param note note.
+   * @param userId user id who triggered the manual reassessment.
+   * @returns the id of the assessment created.
```

Comment:
It actually returns the newly created assessment and not only its ID. Please adjust the comment.